CMakeLists.txt

```diff
@@ -5,10 +5,9 @@ project(roboteam_robothub)
 set(CMAKE_EXPORT_COMPILE_COMMANDS 1)
 SET(CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake_modules")
 
-#set(CMAKE_AUTOMOC ON)
-#set(CMAKE_INCLUDE_CURRENT_DIR ON)
+include(ccache)
```

Comment:
We can do this once in the top-level cmake.

Reply:
Fair enough we should yeah.